go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
 	github.com/google/go-cmp v0.6.0
+	github.com/google/go-licenses v1.6.1-0.20230903011517-706b9c60edd4
```

Comment:
any reason to use hash instead of tag version (e.g. v1.6.0) ?

Reply:
Oh yes I should have mentioned that. It's because of breaking changes, [v2 is around the corner](https://github.com/google/go-licenses/issues/205), so I figured it'd be easier to just change the import path when v2 is out rather than adapt the code.